src/components/users/ResetPasswordForm.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Loader from 'react-loader-spinner';
+import { object, string } from 'prop-types';
+
+import { userActions } from '../../actions/user.actions';
+import '../../style/App.scss';
+import './user-form.scss';
+import FormInput from '../common/FormInput';
+import { userRequest } from '../../actions/user.actions';
+
+const ResetPasswordForm = ({ intl, url_uid, url_token }) => {
+  const dispatch = useDispatch();
+  const [inputs, setInputs] = useState({
+    new_password1: '',
+    new_password2: '',
+  });
+  const { new_password1, new_password2 } = inputs;
+
+  const [isSubmitted, setIsSubmitted] = useState(false);
+
+  const equalPasswords = new_password1 === new_password2;
```

Comment:
What about to modularize validations? In general you will be using almost the same ones along the project (equal, presence , etc). Just to take into consideration for future projects, there are a handful of libraries to handles form validations like [validate js](https://validatejs.org/).

Reply:
I will change this in other PR